Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/serialization_test.go

```diff
@@ -134,17 +136,19 @@ func TestRoundTripTypes(t *testing.T) {
 	// defer api.Scheme.Log(nil)
 
 	for kind := range api.Scheme.KnownTypes("") {
-		if nonRoundTrippableTypes.Has(kind) {
-			continue
-		}
-		// Try a few times, since runTest uses random values.
-		for i := 0; i < *fuzzIters; i++ {
-			doRoundTripTest(kind, t)
+		for version, codec := range map[string]runtime.Codec{"": api.Codec, "v1beta3": v1beta3.Codec, "v1": v1.Codec} {
```

Comment:
Given the way that UPSTREAM likes to test by re-running the whole bucket with a different version, I don't think we'll be able to get the structure of this changed moved up.

What about updating the `serialization_test.go` in our `pkg/api` to not exclude kube types?